### PR TITLE
Practice

### PR DESCRIPTION
**Pull request by phpslonuk**
# Hooli-style_Popup Done please confirm request

### task name

[Demo](https://phpslonuk.github.io/Hooli-style_Popup) |

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/phpslonuk/Popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+.popup-item:hover {
+  outline-style: none;
+  border: 1px solid gray
+}
+
+.popup-item:focus {
+  outline-style: none;
+  border: 1px solid blue
```
Comment (miroslav-kolomiets) [inline submissions/phpslonuk/Popup/css/style.css:141]:
```suggestion
  border-color: blue;
```

File: submissions/phpslonuk/Popup/css/style.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding: 40px 20px 5px 20px;
+  width: 100%;
+  height: 90%
+}
+
+.popup-item:hover {
+  outline-style: none;
+  border: 1px solid gray
```
Comment (miroslav-kolomiets) [inline submissions/phpslonuk/Popup/css/style.css:136]:
```suggestion
  border-color: gray;
```

File: submissions/phpslonuk/Popup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+header {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: black
```
Comment (miroslav-kolomiets) [inline submissions/phpslonuk/Popup/css/style.css:21]: Here (and in other places) add a semicolon `;`

File: submissions/phpslonuk/Popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+.popup-item {
+  width: 80px;
+  height: 80px;
+  padding: 3px;
+  display: flex;
+  justify-content: center;
+  text-align: center
+}
```
[the remark this answers is not in the document]
Comment (miroslav-kolomiets) [reply submissions/phpslonuk/Popup/css/style.css:151]: You have bug with item shifting on hover. You can fix it this way.
Снимок экрана 2020-11-14 в 2 58 37 PM

File: submissions/phpslonuk/Popup/css/style.css [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+  align-content: space-between;
+  flex-wrap: wrap;
+  padding: 40px 20px 10px 20px;
+  width: 100%;
+  height: 55%
+}
+
+.kottans {
+  margin: 10px auto 0px auto;
+}
```
Comment (miroslav-kolomiets) [inline submissions/phpslonuk/Popup/css/style.css:208]: Add new line in the end of the document.

File: submissions/phpslonuk/Popup/index.html [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+                alt="avatar">
+            </a>
+          </li>
+        </ul>
+      </div>
+    </header>
+  </div>
+</body>
+
+</html>
```
Comment (miroslav-kolomiets) [inline submissions/phpslonuk/Popup/index.html:200]: Add new line in the end of the document.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Nice job! But something can be improved.
Make menu items in one color and style please.
Снимок экрана 2020-11-14 в 3 04 42 PM

**Comment by phpslonuk** [comment]
Fixed:
- ";" was adderd.
- added new line in the end of the document.
- top menu improved

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Good job! But you forgot about "Make menu items in one color and style please." that I wrote in first comment.

**Comment by phpslonuk** [comment]
please explain what you mean by one color and style?
Menu

**Comment by miroslav-kolomiets** [comment]
> please explain what you mean by one color and style?
> Menu

Yes, of course, please see the screen:
Снимок экрана 2020-11-16 в 10 48 16 PM

**Comment by miroslav-kolomiets** [comment]
> please explain what you mean by one color and style?
> Menu

Yes, of course, please see the screen:
Снимок экрана 2020-11-16 в 10 53 53 PM

**Comment by phpslonuk** [comment]
I changed the color, added hover

File: submissions/phpslonuk/Popup/css/style.css [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+.nav_item_img {
+  width: 20px;
+  height: 20px;
+  margin-top: 3px;
+  border-radius: 10px;
+}
+
+#trigger {
```
Comment (A-Ostrovnyy) [inline submissions/phpslonuk/Popup/css/style.css:67]: Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkbox in the right way

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Comment by phpslonuk** [comment]
the checkbox was hidden, the right way

File: submissions/phpslonuk/Popup/css/style.css [hunk 8]
```diff
@@ -65,7 +65,8 @@ a {
 }
 
 #trigger {
-  opacity: 0;
+  -webkit-appearance: none;
+  -moz-appearance: none;
```
Comment (A-Ostrovnyy) [inline submissions/phpslonuk/Popup/css/style.css:69]: For future situations after properties with vendor prefixes add the same property without vendor prefixes.)

**Review by A-Ostrovnyy — APPROVED** [review]
LGMT
Well done.

**Comment by A-Ostrovnyy** [comment]
@phpslonuk Why you close PR?

**Comment by suchov** [comment]
probably some miss-understanding here, @phpslonuk pls open the PR one more time, all looks good to us.

**Pull request closed without merging**